util/topgen.py

```diff
@@ -983,8 +985,12 @@ def main():
 
     for ip in generated_list:
         log.info("Appending {}".format(ip))
-        ip_hjson = hjson_dir.parent / "ip/{}/data/autogen/{}.hjson".format(
-            ip, ip)
+        if ip == 'clkmgr':
+            ip_hjson = Path(out_path) / "ip/{}/data/autogen/{}.hjson".format(
```

Comment:
do you mind reminding me why specifically for `clkmgr` this needs to be handled differently?

Reply:
That's because `clkmgr` is the only IP that is generated **before** the initial validation step (see Line 976).

So far, this did not really matter as for `top_earlgrey` we 1) have the auto-generated hjson checked in and 2) the output path and the directory of the hjson config are in the same hierarchy.